krun.py

```diff
@@ -246,6 +247,12 @@ def main(parser):
         if rc != 0:
             util.fatal("Could not touch config file: " + args.filename)
 
+        if not args.develop and not args.dry_run:
+            info(("Wait %s secs to allow system to cool prior to "
```

Comment:
In the scheduler, `--dry_run` does not wait N seconds after rebooting (or simulating a reboot) but it does print "SIMULATED: time.sleep (would have waited %s seconds)." to STDOUT. I think we should be consistent and print something similar here.